### PR TITLE
ScrollView added in LoginActivity for user ease

### PR DESCRIPTION
**Pull request by ishaandhamija**

### CODE REVIEW

File: Android/app/src/main/res/layout/activity_login.xml [hunk 1]
```diff
@@ -10,6 +10,10 @@
     android:paddingTop="@dimen/activity_vertical_margin"
     tools:context=".LoginActivity">
 
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
     <LinearLayout
```
Comment (Swati4star) [inline Android/app/src/main/res/layout/activity_login.xml]: Please maintain the indentation level.
Indent the content between ScrollView opening & closing tags.

**Comment by ishaandhamija** [comment]
Ok Ma'am :)

**Comment by ishaandhamija** [comment]
Done 👍

**Comment by ishaandhamija** [comment]
Please consider this PR ahead :)

**Comment by Swati4star** [comment]
Here also, stash the commits into single one.

**Comment by ishaandhamija** [comment]
Some problem in squashing.. Please close the PR. I will open a new PR with a single commit !

**Comment by Swati4star** [comment]
Cool. I am closing it.

**Pull request closed without merging**